frappe/core/doctype/activity_log/activity_log.py

```diff
@@ -36,8 +36,6 @@ def clear_old_logs(days=None):
 def on_doctype_update():
 	"""Add indexes in `tabActivity Log`"""
 	frappe.db.add_index("Activity Log", ["reference_doctype", "reference_name"])
-	frappe.db.add_index("Activity Log", ["timeline_doctype", "timeline_name"])
```

Comment:
This is used here:
https://github.com/frappe/frappe/blob/63e760e3adba5549bdc0d7146ede1f28820a85de/frappe/model/delete_doc.py#L382

Reply:
Yeah, this will likely do a full table scan removed.


Why do we do this in first place though 🥲